app/javascript/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Greetings from './components/greetings';
```

Comment:
I noticed that you are not fetching the `Greeting` component correctly in `App.js`. 

![image](https://github.com/EddyShimwa/hello-rails-react/assets/38088818/0b5f47fd-3d39-40ed-a51e-26fa5705a274)

- The `Greeting` component is imported as `greetings` yet the file name is `Greeting.js`. Note: using the correct naming conventions is very crucial when importing components to make sure that you are importing the correct component.